### ISSUE TITLE
[feature]: Add scale functionality to text

### ISSUE THREAD

**Issue by BenJamesBen** [freesewing/freesewing#2832]
### What is it that you would like to see happen? 🤔

Add functionality to allow text added via the `data-text` attribute to be scaled. Perhaps via the `data-scale` attribute or a new `data-text-scale` attribute.

### Are you a FreeSewing patron? 😃

No, I am not 😞

### Additional context

This functionality would be used mostly for doll-sized patterns.

A workaround might be to have the design code determine what CSS class should be used for text size, and set it via the `data-text-class` attribute.

I note that the text scale can be changed by adding a `data-text-transform` attribute.
`attr('data-text-transform', 'scale(0.5 0.5)')` does work, but it has the side effect of moving the text away from the anchor.

Using a matrix transform can be used to move the scaled text back to the anchor.
```js
function getScaleMatrix(anchor, scale) {
   const cx = anchor.x - scale * anchor.x
   const cy = anchor.y - scale * anchor.y
 
   return `matrix(${scale}, 0, 0, ${scale}, ${cx}, ${cy})`
 }

      points.bottomCenterBack = new Point(0, points.bottomLeftBack.y)
         .attr('data-text', 'Back bottom')
         .attr('data-text-class', 'left fill-note')
       points.bottomCenterBack
         .attr('data-text-transform',
           getScaleMatrix(points.bottomCenterFront, textscale))
```

**Comment by woutervdub** [comment]
Is this still an issue? I think this has been resolved with #3902

**Comment by BenJamesBen** [comment]
Closing as a duplicate of/superceded by #3902.
